vm/src/class.rs

```diff
@@ -112,6 +112,16 @@ pub trait PyClassImpl: PyClassDef {
                     .into();
             class.set_attr(identifier!(ctx, __new__), bound);
         }
+
+        if class
+            .slots
+            .hash
+            .load()
+            .map(|h| h as usize == unhashable_wrapper as usize)
+            .unwrap_or(false)
```

Comment:
```suggestion
            .map_or(0, |h| h as usize) == unhashable_wrapper as usize
```
the cost of `map_or(0, |h| h as usize)` is zero here due to its layout.